h. AUGUST.md

```diff
@@ -4,4 +4,4 @@
 | Microsoft Diversity                                 | not sure      | Scholarship                                               | [https://careers.microsoft.com/students/us/en/usscholarshipprogram](https://careers.microsoft.com/students/us/en/usscholarshipprogram)                                                                                                 |
 | Mozilla Fellowship                                  | not sure      | Fellowship                                                | [https://blog.mozilla.org/blog/2019/03/12/apply-for-a-mozilla-fellowship/](https://blog.mozilla.org/blog/2019/03/12/apply-for-a-mozilla-fellowship/)                                                                                   |
 | Hack@Home by Atlassian                              |               | Internship                                                | [https://www.linkedin.com/pulse/all-atlassian-hackhome-coding-challenge-lakshmi-k-p/](https://www.linkedin.com/pulse/all-atlassian-hackhome-coding-challenge-lakshmi-k-p/)                                                             |
-|                                                     |               |                                                           |                                                                                                                                                                                                                                        |
+|  Github Campus EXpert applications open  |      not sure         |   Campus Expert Program                                      |  https://education.github.com/experts                                                                                                                                                                                             |
```

Comment:
"EXpert" to "Expert". 

And it happens twice every year. 1st week of March and August. But I would suggest you verify from any Github Campus Expert in your connection.

Reply:
Yes it happens twice! I will correct the typo. Thank you!